build.fsx

```diff
@@ -134,7 +139,7 @@ Target.Create "RenameFSharpCompilerService" (fun _ ->
     for framework in ["netstandard1.6"; "net45"] do
       let dir = __SOURCE_DIRECTORY__ </> "packages"</>packDir</>"lib"</>framework
       let targetFile = dir </>  "FAKE.FSharp.Compiler.Service.dll"
-      File.DeleteFile targetFile
+      File.Delete targetFile
```

Comment:
This is System.IO.File.Delete isn't it?
Should we provide `delete` (ie lower case version) to have unified looking code?

I think there is no easy answer.

Reply:
yes that's it. I saw System.IO.File is used a lot because some functions in Fake.IO.FileSystem modules are just one line call to BCL...
I will replace it in build.fsx